<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeViewer2D</class>
 <widget class="QMainWindow" name="VolumeViewer2D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VolumeViewer2D</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="VolumeView" name="volumeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
      </widget>
      <widget class="QMdiArea" name="mdiArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="mouseToolBar">
   <property name="windowTitle">
    <string>Mouse Modes</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Standard</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_Player"/>
   <addaction name="actionBack"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Display">
    <property name="title">
     <string>&amp;Display</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSet_Marker_Color"/>
    <addaction name="actionSet_Table_Color"/>
    <addaction name="actionSet_Last_Viewed_Color"/>
    <addaction name="separator"/>
    <addaction name="actionDisplay_Horizons"/>
    <addaction name="actionDisplay_Wells"/>
    <addaction name="actionDisplay_Markers"/>
    <addaction name="actionDisplay_Marker_Labels"/>
    <addaction name="actionDisplay_Tables"/>
    <addaction name="actionDisplay_Last_Viewed"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Items">
    <property name="title">
     <string>&amp;Items</string>
    </property>
    <addaction name="actionSetup_Volumes"/>
    <addaction name="actionSetup_Horizons"/>
    <addaction name="actionSetup_Wells"/>
    <addaction name="actionSetup_Tops"/>
    <addaction name="actionSetup_Tables"/>
   </widget>
   <widget class="QMenu" name="menu_Navigation">
    <property name="title">
     <string>&amp;Navigation</string>
    </property>
    <addaction name="action_Player"/>
    <addaction name="actionBack"/>
   </widget>
   <widget class="QMenu" name="menuPicking">
    <property name="title">
     <string>Pic&amp;king</string>
    </property>
    <addaction name="action_New_Picks"/>
    <addaction name="action_Load_Picks"/>
    <addaction name="action_Save_Picks"/>
    <addaction name="actionSave_As_Picks"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Items"/>
   <addaction name="menu_Display"/>
   <addaction name="menu_Navigation"/>
   <addaction name="menuPicking"/>
   <addaction name="menu_Window"/>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionDisplay_Range">
   <property name="icon">
    <iconset resource="../ProjectViewer/icons.qrc">
     <normaloff>:/icons/images/select-range-icon-32x32.png</normaloff>:/icons/images/select-range-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>Volume Display &amp;Range</string>
   </property>
  </action>
  <action name="actionColorbar">
   <property name="icon">
    <iconset resource="../ProjectViewer/icons.qrc">
     <normaloff>:/icons/images/colortable-icon-32x32.png</normaloff>:/icons/images/colortable-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>Vol&amp;ume Colortable</string>
   </property>
  </action>
  <action name="action_Sampling">
   <property name="text">
    <string>Edit &amp;Sampling</string>
   </property>
  </action>
  <action name="actionChange_Domain">
   <property name="text">
    <string>Change &amp;Domain</string>
   </property>
  </action>
  <action name="actionChanged_Type">
   <property name="text">
    <string>Changed &amp;Type</string>
   </property>
  </action>
  <action name="actionSetup_Horizons">
   <property name="text">
    <string>Setup &amp;Horizons</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionSet_Horizon_Color">
   <property name="text">
    <string>Set &amp;Horizon Color</string>
   </property>
  </action>
  <action name="actionSetup_Wells">
   <property name="text">
    <string>Setup &amp;Wells</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSet_Well_Color">
   <property name="text">
    <string>Set &amp;Well Color</string>
   </property>
  </action>
  <action name="actionSet_Last_Viewed_Color">
   <property name="text">
    <string>Set &amp;Last Viewed Color</string>
   </property>
  </action>
  <action name="actionDisplay_Horizons">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Display Horizons</string>
   </property>
  </action>
  <action name="actionDisplay_Wells">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>D&amp;isplay Wells</string>
   </property>
  </action>
  <action name="actionDisplay_Last_Viewed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display Last &amp;Viewed</string>
   </property>
  </action>
  <action name="actionSetup_Volumes">
   <property name="text">
    <string>Setup &amp;Volumes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionVolume_Opacity">
   <property name="text">
    <string>Volume &amp;Opacity</string>
   </property>
  </action>
  <action name="actionVolume_Z_Value">
   <property name="text">
    <string>Volume &amp;Z-Value</string>
   </property>
  </action>
  <action name="action_Configure_Volumes">
   <property name="text">
    <string>&amp;Configure Volumes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Player">
   <property name="icon">
    <iconset resource="../ProjectViewer/icons.qrc">
     <normaloff>:/icons/images/play-32x32.png</normaloff>:/icons/images/play-32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Player</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="../ProjectViewer/icons.qrc">
     <normaloff>:/icons/images/back-32x32.png</normaloff>:/icons/images/back-32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Back</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionDisplay_Markers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display &amp;Tops</string>
   </property>
  </action>
  <action name="actionSet_Marker_Color">
   <property name="text">
    <string>&amp;Set Tops Color</string>
   </property>
  </action>
  <action name="actionSetup_Tops">
   <property name="text">
    <string>Setup &amp;Tops</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionDisplay_Marker_Labels">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dis&amp;play Top Labels</string>
   </property>
  </action>
  <action name="actionSetup_Tables">
   <property name="text">
    <string>&amp;Setup Tables</string>
   </property>
  </action>
  <action name="actionDisplay_Tables">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display Ta&amp;bles</string>
   </property>
  </action>
  <action name="actionSet_Table_Color">
   <property name="text">
    <string>S&amp;et Table Color</string>
   </property>
  </action>
  <action name="action_Load_Picks">
   <property name="text">
    <string>&amp;Load Picks</string>
   </property>
  </action>
  <action name="action_Save_Picks">
   <property name="text">
    <string>&amp;Save Picks</string>
   </property>
  </action>
  <action name="action_New_Picks">
   <property name="text">
    <string>&amp;New Picks</string>
   </property>
  </action>
  <action name="actionSave_As_Picks">
   <property name="text">
    <string>Save &amp;As Picks </string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VolumeView</class>
   <extends>QGraphicsView</extends>
   <header>volumeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ProjectViewer/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>VolumeViewer2D</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
